fletcher/base.py

```diff
@@ -235,6 +230,27 @@ def size(self):
         """
         return len(self.data)
 
+    def __arrow_array__(self, type=None):
+        # type: (pa.DataType,) -> pa.Array
+        """
+        Implement pyarrow array interface (requires pyarrow>=0.15).
+
+        Returns
+        -------
+        pa.Array
+
+        """
+        if self._has_single_chunk:
+            data = self.data.chunks[0]
+        else:
+            data = pa.concat_arrays(self.data.iterchunks())
+            self.data = pa.chunked_array([data])  # modify a data pointer inplace
```

Comment:
you need to make `offsets`  property otherwise it's going to be misaligned. Probably, it's better to add `__arrow_array__` in dedicated PR.

Reply:
You're right, I removed it for this PR.